### ISSUE TITLE
Why doesn't vcpkg.io Browse Packages=>View Detail not show me which `target_link_libraries()` are made available?

### ISSUE THREAD

**Issue by ned14** [microsoft/vcpkg#18051]
It seems silly to me that the website shows `vcpkg install pkg` in the View Detail, but despite that vcpkg knows what cmake targets are exported for linking, vcpkg.io doesn't list them. This forces people to go install the thing on the command line just to get it to print what cmake targets are available. This makes it harder than necessary to port a project from another package manager, where cmake targets are named slightly different to vcpkg, to vcpkg.

**Comment by JackBoosY** [comment]
This is because some usage is dynamically generated when building the port.

**Comment by ned14** [comment]
> This is because some usage is dynamically generated when building the port.

True. However, one could cache that data, and then when one updates the versions tree, one could store there the cmake targets which were dynamically generated when building the port before the version was updated.

One thing I've noticed during a port from cmake hunter is that many vcpkg libraries don't support static library editions of themselves in vcpkg, even though the exact same package does so in cmake hunter. If the vcpkg.io summary also could show that, then we would be really swimming.

**Comment by dg0yt** [comment]
The dynamically generated information doesn't always represent canonical usage.
Some ports do have static information in a `usage` file, e.g. `fmt`. However, IIUC only the installed file name is standardized, not the name in the port recipe.

**Comment by JackBoosY** [comment]
Will be partly fixed by https://github.com/microsoft/vcpkg-tool/pull/82. But our webset still don't show the usage since you need to install the port before use it.
